apps/easypid/src/features/share/components/RequestedAttributesSection.tsx

```diff
@@ -14,71 +13,70 @@ export type RequestedAttributesSectionProps = {
-                    name={credential.credentialName}
-                    backgroundImage={credential.backgroundImage}
-                    backgroundColor={credential.backgroundColor}
-                    issuerImage={credential.issuerImage}
-                    textColor={credential.textColor}
-                    disclosedAttributes={credential.requestedAttributes ?? []}
-                    disclosedPayload={credential?.disclosedPayload ?? {}}
-                    isExpired={
-                      credential.metadata?.validUntil ? new Date(credential.metadata.validUntil) < new Date() : false
-                    }
-                    isNotYetActive={
-                      credential.metadata?.validFrom ? new Date(credential.metadata.validFrom) > new Date() : false
-                    }
-                  />
-                )
-              })}
-            </YStack>
-          ))}
         </YStack>
+        {/* We always take the first one for now (no selection) */}
+        {satisfiedEntries.map(({ credentials: [credential], ...entry }) => {
+          return (
+            <CardWithAttributes
+              key={entry.inputDescriptorId}
+              id={credential.credential.id}
+              name={credential.credential.display.name}
+              backgroundImage={credential.credential.display.backgroundImage}
+              backgroundColor={credential.credential.display.backgroundColor}
+              issuerImage={credential.credential.display.issuer.logo}
+              textColor={credential.credential.display.textColor}
+              formattedDisclosedAttributes={getDisclosedAttributeNamesForDisplay(credential)}
+              disclosedPayload={credential.disclosed.attributes}
+              isExpired={
+                credential.credential.metadata?.validUntil
+                  ? new Date(credential.credential.metadata.validUntil) < new Date()
+                  : false
+              }
+              isNotYetActive={
+                credential.credential.metadata?.validFrom
+                  ? new Date(credential.credential.metadata.validFrom) > new Date()
+                  : false
+              }
+            />
+          )
+        })}
+        {unsatisfiedEntries.length > 0 && (
+          <>
+            {satisfiedEntries.length !== 0 && (
+              <YStack>
+                <Heading variant="sub2">UNAVAILABLE CARDS</Heading>
+              </YStack>
+            )}
+            {unsatisfiedEntries.map((entry) => (
+              <CardWithAttributes
+                key={entry.inputDescriptorId}
+                name={entry.name ?? 'Credential'}
+                // We only have the attribute paths, no way to know how to render
+                // TODO: we could look at the vct?
+                // TODO: we should maybe support partial matches (i.e. vct matches), as then we can
+                // show a much better UI (you have the cred, but age is not valid, or this param is missing)
```

Comment:
Do you mean then rendering the credential card but with 'date_of_birth' in red when it doesn't match the required age?